kuzzleio/kuzzle-build
# update setup grammar

## What does this PR do ?
Correct Grammar of Setup.sh

### How should this be manually tested?
Read it over
And run the setup.sh script to make sure it still works and looks ok.
